doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -6,7 +6,7 @@
 
 **X-ROAD 6**
 
-Version: 2.46
+Version: 2.47
```

Comment:
Spaces

Reply:
Should come from develop branch actually -> develop should probably be merged to this branch

Reply:
Merged and fixed